src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageStrings.cs

```diff
@@ -13,18 +14,33 @@ internal static class AdvancedOptionPageStrings
         public static string Option_Analysis
             => ServicesVSResources.Analysis;
 
-        public static string Option_Background_analysis_scope
-            => ServicesVSResources.Background_analysis_scope_colon;
+        public static string Option_Run_background_code_analysis_for
+            => ServicesVSResources.Run_background_code_analysis_for_colon;
+
+        public static string Option_Background_Analysis_Scope_None
+            => ServicesVSResources.None;
 
         public static string Option_Background_Analysis_Scope_Active_File
             => ServicesVSResources.Current_document;
 
-        public static string Option_Background_Analysis_Scope_Open_Files_And_Projects
+        public static string Option_Background_Analysis_Scope_Open_Files
             => ServicesVSResources.Open_documents;
 
         public static string Option_Background_Analysis_Scope_Full_Solution
             => ServicesVSResources.Entire_solution;
 
+        public static BackgroundAnalysisScope Option_Background_Analysis_Scope_None_Tag
+            => BackgroundAnalysisScope.None;
+
+        public static BackgroundAnalysisScope Option_Background_Analysis_Scope_Active_File_Tag
+            => BackgroundAnalysisScope.ActiveFile;
+
+        public static BackgroundAnalysisScope Option_Background_Analysis_Scope_Open_Files_Tag
+            => BackgroundAnalysisScope.OpenFiles;
+
+        public static BackgroundAnalysisScope Option_Background_Analysis_Scope_Full_Solution_Tag
```

Comment:
this feels funky.  everything else in this file is just strings right... why are these enum options going here?  i'm not quite sure i get it...